test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -187,7 +187,7 @@ public void MulticlassSdcaTest()
         /// <summary>
         /// Multiclass Logistic Regression test with a tree featurizer.
         /// </summary>
-        [X64Fact("x86 output differs from Baseline")]
```

Comment:
>X64Fact [](start = 9, length = 7)

can we remove X64Fact now?

Reply:
As @mstfbl pointed out above, I think there are still around 4 tests that are causing test process crashes in x86. They need investigation.

Reply:
I see, thanks

---
In reply to: [414314991](https://github.com/dotnet/machinelearning/pull/5057#discussion_r414314991) [](ancestors = 414314991)